test/cholmod.jl

```diff
@@ -281,6 +284,37 @@ end
     end
 end
 
+@testset "ldiv! $Tv $Ti" begin
```

Comment:
Does this test Dense input?

Reply:
This tests standard dense Julia `Vector`s as input right-hand sides. `Dense` structs are tested above in the new lines 141-143.